config/crd/bases/kafka.nais.io_aivenapplications.yaml

```diff
@@ -0,0 +1,107 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.5
+  creationTimestamp: null
+  name: aivenapplications.kafka.nais.io
+spec:
+  additionalPrinterColumns:
+  - JSONPath: .metadata.creationTimestamp
+    name: Age
```

Comment:
I added this because we have it on other CRDs, but I'm not sure how useful it is. I think maybe reordering to put the secretname and the expiration timestamps up front, then sync state and finally age. What do you think?